Core/Python/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I don't think any of these files are required / should have made it into the push to the repo

Reply:
This was already fixed in one of the submitted commits